Src/Support/Google.Apis.Auth/OAuth2/ExternalAccountCredential.cs

```diff
@@ -262,7 +275,19 @@ static string ExtractTargetPrincipal(string url)
         /// <summary>
         /// Gets the subject token to be exchanged for the access token.
         /// </summary>
-        protected abstract Task<string> GetSubjectTokenAsync(CancellationToken taskCancellationToken);
+        protected abstract Task<string> GetSubjectTokenAsyncImpl(CancellationToken taskCancellationToken);
+
+        private async Task<string> GetSubjectTokenAsync(CancellationToken taskCancellationTokne)
+        {
+            try
+            {
+                return await GetSubjectTokenAsyncImpl(taskCancellationTokne).ConfigureAwait(false);
+            }
+            catch(Exception ex)
```

Comment:
Micro-nit: space before `(`

Reply:
Done. I'll rebase all and merge after green! Thanks.